examples/example_b2c/lib/main.dart

```diff
@@ -0,0 +1,154 @@
+import 'package:aad_oauth/aad_oauth.dart';
+import 'package:aad_oauth/model/config.dart';
+import 'package:flutter/material.dart';
+
+void main() => runApp(new MyApp());
+
+class MyApp extends StatefulWidget {
+  @override
+  _MyAppState createState() => new _MyAppState();
+}
+
+class _MyAppState extends State<MyApp> {
+  @override
+  initState() {
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return new MaterialApp(
+      title: 'AAD B2C Demo',
+      theme: new ThemeData(
+        primarySwatch: Colors.blue,
+      ),
+      home: new MyHomePage(title: 'AAD B2C Home'),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
+  MyHomePage({Key key, this.title}) : super(key: key);
+  final String title;
+
+  @override
+  _MyHomePageState createState() => new _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+
+  static final Config configB2Ca = new Config(
+    "YOUR_TENANT_ID",
+    "YOUR_CLIENT_ID",
+    "YOUR_CLIENT_ID offline_access",
+    "https://login.live.com/oauth20_desktop.srf",
+    clientSecret: "YOUR_CLIENT_SECRET",
+    isB2C: true,
+    azureTenantName: "YOUR_TENANT_NAME",
+    userFlow: "YOUR_USER_FLOW___USER_FLOW_A",
```

Comment:
can you add 2 different tokenIdentifiers to configB2Ca and configB2Cb? then I can merge it